<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address list</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="createField">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>201</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="nameField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="timeField">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>111</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>201</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Creation time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Timeout</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QListWidget" name="addressField">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>111</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
